content/tutorial/tutorial.md

```diff
@@ -1042,20 +1049,21 @@ Let's `map` over the `history` in the Game's `render` method:
   }
 ```
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
 
-For each move in the tic-tac-toes's game's history, we create a list item `<li>` which contains a button `<button>`. The button has a `onClick` handler which calls a method called `this.jumpTo()`. We haven't implemented the `jumpTo()` method yet. For now, we should see a list of the moves that have occurred in the game and a warning in the developer tools console that says:
+टिक-टैक्-टो के खेल के इतिहास में प्रत्येक चाल के लिए, हम एक सूची आइटम `<li>` बनाते हैं जिसमें एक बटन होता है `<button>`। बटन में एक  `onClick` हैंडलर होता है, जो कि `this.jumpTo()` नामक एक मेथड कॉल करता है। 
+हमने अभी तक `jumpTo()` मेथड बनाई नहीं की है। अभी के लिए, हमें खेल में होने वाली चालों की सूची दिखनी चाहिए और डेवलपर टूल्स कंसोल में एक चेतावनी दी गई है जो कहती है:
 
->  Warning:
->  Each child in an array or iterator should have a unique "key" prop. Check the render method of "Game".
+>  चेतावनी:
+>  एक array या पुनरावृत्त में प्रत्येक चाइल्ड के पास एक अद्वितीय "कुंजी" प्रोप होना चाहिए। "Game" की रेंडर मेथड की जाँच करें।
```

Comment:
iterator should not be translated?

Reply:
"Game" की रेंडर मेथड की जाँच करें। -> "Game" ke रेंडर मेथड की जाँच करें।

Reply:
> iterator

Yes. sounds good. Hindi meaning sounds a bit awkward